test/csuite/timestamp_abort/main.c

```diff
@@ -434,9 +436,23 @@ timestamp_build(void)
 extern int __wt_optind;
 extern char *__wt_optarg;
 
+static void
+handler(int sig)
+{
+	pid_t pid;
+
+	WT_UNUSED(sig);
+	pid = wait(NULL);
+	/*
+	 * The core file will indicate why the child exited. Choose EINVAL here.
+	 */
+	testutil_die(EINVAL, "Child process %d abnormally exited", pid);
```

Comment:
`pid` isn't necessarily an int, so at least it needs casting, and I'd go with a  `u_int` (and `uintmax_t` is for the rigorous).

Reply:
Why bother waiting, do we care about the child pid, given that it just exited?

Reply:
If the parent doesn't call `wait` then the child process remains a zombie.  I'll fix the output message.

Reply:
> then the child process remains a zombie. I'll fix the output message.

How is that possible, doesn't the init(8) process (or whatever passes for init these days) eventually do the wait?

Reply:
The init process would become the parent if the parent process exits.  But the child process remains a zombie upon exit so that the parent can call wait to get its exit status.  If this parent process knows the child has exited, I think it should call wait/waitpid so the child can be properly cleaned up.